models/GroupUserDetails.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+    const GroupUserDetails = sequelize.define('GroupUserDetails', {
+
+
+        name: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            unique: true,
```

Comment:
let's remove `unique` since that'll prevent people in different groups from having the same name